dockers/docker-fpm-frr/zsocket.sh

```diff
@@ -23,11 +23,10 @@ while getopts ":a:o:h" opt; do
             ;;
     esac
 done
-shift $((OPTIND-1))
 
 timeout 5s bash -c -- "until </dev/tcp/${addr}/${port}; do sleep 0.1;done"
 if [ "$?" != "0" ]; then
     logger -p error "Error: zebra is not ready to accept connections"
 fi
```

Comment:
even if return value is zero, suggest to logger -p info for the time spend on waiting, it will be helpful for future debugging purpose.

Reply:
Added an info logger to log the time spent in waiting.